website/docs/core-concepts/writing-code/ext-libraries.md

```diff
@@ -1,26 +1,61 @@
 
-## JavaScript Library Reference
+ <VideoEmbed host="youtube" videoId="tqJna718tj4" title="Using Built-in Libraries in Appsmith" caption="Using Built-in Libraries in Appsmith"/> 
 
-* [lodash](https://lodash.com/docs/4.17.15)
-* [moment](https://momentjs.com/docs/)
-* [btoa](https://github.com/dankogai/js-base64#readme)
-* [atob](https://github.com/dankogai/js-base64#readme)
-* [xmlParser](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v3/docs.md)
-* [Forge](https://github.com/digitalbazaar/forge)
 
- <VideoEmbed host="youtube" videoId="tqJna718tj4" title="Using Built-in Libraries in Appsmith" caption="Using Built-in Libraries in Appsmith"/> 
+## Pre-installed JavaScript libraries
+
+There are variety of pre-installed JavaScript libraries that can be utilized in your projects. These libraries provide access to a range of useful functions and can be incorporated into your applications.
+
+* [Lodash](https://lodash.com/docs/4.17.15), is a JavaScript utility library that provides functions for common programming tasks such as formatting data, iterating over collections, and manipulating arrays and objects.
+
+* [Moment](https://momentjs.com/docs/), is a JavaScript library that simplifies working with dates and times in JavaScript by providing functions for parsing, validating, manipulating, and displaying dates and times.
+
+* [xmlParser](https://naturalintelligence.github.io/fast-xml-parser/), is a library for parsing and manipulating XML data in JavaScript.
+
+* [Forge](https://github.com/digitalbazaar/forge), is a library for working with cryptographic algorithms and protocols in JavaScript.
+
+
+## Install custom JavaScript libraries
+
+You can browse and install recommended JS libraries or install a JS library of your choice by providing a valid URL. 
 
+For example- 
+
+*  Valid and supported:  `https://cdn.jsdelivr.net/npm/exceljs@4.3.0/dist/exceljs.min.js`. 
+
+* Invalid URL: `https://www.jsdelivr.com/package/npm/datejs`
+
+* Valid URL but not supported: `https://cdn.jsdelivr.net/npm/uuid@9.0.0/dist/index.js`
+
+You can search for libraries on popular CDN services like [jsDelivr](https://www.jsdelivr.com/) or [UNPKG](https://unpkg.com/) to find the URL, which points to the library's index file. It's important to ensure that the library supports a [UMD](https://github.com/umdjs/umd) build to work with Appsmith. For more information, see the [basic pattern](https://github.com/umdjs/umd/blob/master/templates/commonjsStrict.js) of a UMD build.
+
+Many libraries may have a `.min.js` file located in the `root`, `/umd`, or `/browser` folders. If the library you wish to use doesn't support a UMD build, you can use `browserify` to create one and host it on a CDN of your choice.
+
```

Comment:
Let there be a subheading under this called **Install external library**

Reply:
added a sub head